u4-databases/features/uni-controller.js

```diff
@@ -0,0 +1,54 @@
+const { db } = require('../_services/firebase-initialized')
+
+const iterationLists = async (req, res) => {
+  const query = db.collection('iterations').get()
+  const iteration = (await query).docs.map(doc => doc.data())
+
+  res.render('uni-iteration-list', { iteration })
+}
+
+const iterationDetails = async (req, res) => {
+  const code = req.params.code
+  const query = db.collection('iterations').doc(code).get()
+  const iteration = (await query).data()
+
+  res.render('uni-iteration-details', { iteration })
```

Comment:
ตัวแปรนี้เป็นอ็อบเจ็กต์การวนซ้ำเดียว ชื่อตัวแปรเอกพจน์ของคุณ `iteration` ดีที่นี่!
This variable is a single iteration object. Your singular variable name `iteration` is good here!